### PR TITLE
eBPF proc fallback

### PR DESCRIPTION
**Pull request by schu**
Not ready for merge yet, requires

* [x] rebase when #2326 is merged +
* [x] a test.

### CODE REVIEW

**Comment by schu** [comment]
Updated.

**Comment by schu** [comment]
Fixes #2215

File: probe/endpoint/connection_tracker.go [hunk 1]
```diff
@@ -28,43 +29,37 @@ type connectionTracker struct {
...
-		noopConnectionTracker := connectionTracker{
-			conf:            conf,
-			flowWalker:      nil,
-			ebpfTracker:     nil,
-			reverseResolver: nil,
-		}
-		return noopConnectionTracker
+		// ebpf failed, fallback to conntrack for connection tracking
+		log.Warnf("Error setting up the eBPF tracker: %v", err)
+		log.Warnf("Fallback to proc scanning")
```
Comment (2opremio) [inline probe/endpoint/connection_tracker.go]: `Falling back to`, also, please use a single log-line

**Comment by 2opremio** [comment]
LGT minor-comment side

File: probe/endpoint/connection_tracker.go (hunk 1 above)
Comment (schu) [inline probe/endpoint/connection_tracker.go]: Done.

**Pull request merged**